src/radical/pilot/agent/agent_0.py

```diff
@@ -340,77 +350,59 @@ def _start_services(self):
         If a `./services` file exist, reserve a compute node and run that file
         there as bash script.
         '''
+        self._log.info("Starting the agent services")
+        service_descriptions = self._cfg.services
+        services = list()
+
+        for service_desc in service_descriptions:
+            task = dict()
+            task['origin'] = 'services'
+            task['description'] = TaskDescription(service_desc).as_dict()
+            task['state'] = rps.AGENT_STAGING_INPUT_PENDING
+            task['status'] = 'NEW'
+            task['type'] = 'task'
+            task['uid'] = ru.generate_id('services.%(item_counter)04d',
+                                             ru.ID_CUSTOM, ns="TODO_WHATTOUSE")
+            self._service_task_ids.append(task['uid'])
+            task['pilot_sandbox'] = self._cfg.pilot_sandbox
+            task['task_sandbox'] = self._cfg.pilot_sandbox + task['uid'] + '/'
+            task['task_sandbox_path'] = self._cfg.pilot_sandbox + task['uid'] + '/'
+            task['session_sandbox'] = self._cfg.session_sandbox
+            task['resource_sandbox'] = self._cfg.resource_sandbox
+            task['pilot'] = self._cfg.pid
+            # task['resources'] = {'cpu': td['ranks'] *
+            #                             td.get('cores_per_rank', 1),
+            #                      'gpu': td['ranks'] *
+            #                             td.get('gpus_per_rank', 1)}
+            services.append(task)
+        self.number_of_services_to_launch = len(services)
+        self.advance(services, publish=False, push=True)
```

Comment:
please check the coding guidelines for consistent indentation and spacing.

Reply:
got it